docs/code_verification.md

```diff
@@ -17,11 +17,17 @@ mpiexec -n 40 ./ex2b -save true -output_prefix dambreak500x1000 -mesh ../share/m
 ## Run MMS simulations
 
 ```
-mpiexec -n 1 ./ex2b_MMS -savef true -output_prefix MMS_dx1 -mesh ../share/meshes/ex2b_MMS_mesh_dx1.exo -dt 0.01 -ts_max_time 5 -ts_exact_final_time matchstep
-mpiexec -n 2 ./ex2b_MMS -savef true -output_prefix MMS_dx0.5 -mesh ../share/meshes/ex2b_MMS_mesh_dx0.5.exo -dt 0.01 -ts_max_time 5 -ts_exact_final_time matchstep
-mpiexec -n 2 ./ex2b_MMS -savef true -output_prefix MMS_dx0.25 -mesh ../share/meshes/ex2b_MMS_mesh_dx0.25.exo -dt 0.01 -ts_max_time 5 -ts_exact_final_time matchstep
-mpiexec -n 2 ./ex2b_MMS -savef true -output_prefix MMS_dx0.1 -mesh ../share/meshes/ex2b_MMS_mesh_dx0.1.exo -dt 0.01 -ts_max_time 5 -ts_exact_final_time matchstep
-mpiexec -n 4 ./ex2b_MMS -savef true -output_prefix MMS_dx0.05 -mesh ../share/meshes/ex2b_MMS_mesh_dx0.05.exo -dt 0.01 -ts_max_time 5 -ts_exact_final_time matchstep
+suffixes=("1" "0.5" "0.25" "0.1" "0.05")
+for suffix in $suffixes
```

Comment:
I suggest changing to `for suffix in "${suffixes[@]}"` otherwise it does not loop through all elements in this list. 

Other than this, I don't see any other issues and the results look good.